fastergt/src/play/template2/JavaExtensions.java

```diff
@@ -256,16 +258,7 @@ public static String urlEncode(String entity) {
     }
 
     public static String formatSize(Long bytes) {
-        if (bytes < 1024L) {
-            return bytes + " B";
-        }
-        if (bytes < 1048576L) {
-            return bytes / 1024L + "KB";
-        }
-        if (bytes < 1073741824L) {
-            return bytes / 1048576L + "MB";
-        }
-        return bytes / 1073741824L + "GB";
+        return Utils.formatSize(bytes);
```

Comment:
This is used in `framework` so it cannot be moved out. Hence I've put it in Utils.